lib/Targets/AIETargetXAIEV2.cpp

```diff
@@ -938,14 +939,14 @@ mlir::LogicalResult AIETranslateToXAIEV2(ModuleOp module, raw_ostream &output) {
     if (!lock.hasName())
       return;
     std::string lockName(lock.name().getValue());
-    output << "int mlir_aie_acquire_" << lockName << "(" << ctx_p
+    output << "AieRC mlir_aie_acquire_" << lockName << "(" << ctx_p
```

Comment:
Understood, I'll put this back to int. The rationale was that, in the tutorials, this is checked against XAIE_OK. Since it's just a macro, hence untyped, that works with ints too, though.

Reply:
Changed all generated functions in `aie_inc.cpp` to return `int`, so we no longer return `AieRC` anywhere. Note that this changes the signature of more functions than just this one. It should now be consistent.